index.js

```diff
@@ -7,43 +7,12 @@ function ask(questionText) {
   });
 }
 
-function getMedian(maximum, minimum) {
-  return Math.round((maximum + minimum) / 2)
-}
+start();
 
 async function start() {
-
-  let max = 100
-  let min = 1
-
-  let guess = getMedian(max, min)
-
-
-
   console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?");
+  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
-
-  let yesNo = await ask("Is your number" + guess)
-
-  while (yesNo !== 'y') {
-
-    if (guess < secretNumber) {
-      min = guess + 1
-      guess = getMedian(min, max)
-      yesNo = await ask ("Is your secret number the" + guess)
-
-    }
-
-    if (guess > secretNumber) {
```

Comment:
Like above, imagine that you don't have access to `secretNumber` and instead make the computer rely solely on human input.